fsw/src/novatel_oem615_app.c

```diff
@@ -17,67 +17,13 @@
 */
 NOVATEL_OEM615_AppData_t NOVATEL_OEM615_AppData;
 
-static CFE_EVS_BinFilter_t  NOVATEL_OEM615_EventFilters[] =
-{   /* Event ID    mask */
-    {NOVATEL_OEM615_RESERVED_EID,                0x0000},
-    {NOVATEL_OEM615_STARTUP_INF_EID,             0x0000},
-    {NOVATEL_OEM615_LEN_ERR_EID,                 0x0000},
-    {NOVATEL_OEM615_PIPE_ERR_EID,                0x0000},
-    {NOVATEL_OEM615_SUB_CMD_ERR_EID,             0x0000},
-    {NOVATEL_OEM615_SUB_REQ_HK_ERR_EID,          0x0000},
-    {NOVATEL_OEM615_PROCESS_CMD_ERR_EID,         0x0000},
-    {NOVATEL_OEM615_CMD_ERR_EID,                 0x0000},
-    {NOVATEL_OEM615_CMD_NOOP_INF_EID,            0x0000},
-    {NOVATEL_OEM615_CMD_RESET_INF_EID,           0x0000},
-    {NOVATEL_OEM615_CMD_ENABLE_INF_EID,          0x0000},
-    {NOVATEL_OEM615_ENABLE_INF_EID,              0x0000},
-    {NOVATEL_OEM615_ENABLE_ERR_EID,              0x0000},
-    {NOVATEL_OEM615_CMD_DISABLE_INF_EID,         0x0000},
-    {NOVATEL_OEM615_DISABLE_INF_EID,             0x0000},
-    {NOVATEL_OEM615_DISABLE_ERR_EID,             0x0000},
-    {NOVATEL_OEM615_CMD_CONFIG_INF_EID,          0x0000},
-    {NOVATEL_OEM615_CONFIG_INF_EID,              0x0000},
-    {NOVATEL_OEM615_CONFIG_ERR_EID,              0x0000},
-    {NOVATEL_OEM615_DEVICE_TLM_ERR_EID,          0x0000},
-    {NOVATEL_OEM615_REQ_HK_ERR_EID,              0x0000},
-    {NOVATEL_OEM615_REQ_DATA_ERR_EID,            0x0000},
-    {NOVATEL_OEM615_UART_INIT_ERR_EID,           0x0000},
-    {NOVATEL_OEM615_UART_CLOSE_ERR_EID,          0x0000},
-    {NOVATEL_OEM615_UART_READ_ERR_EID,           0x0000},
-    {NOVATEL_OEM615_UART_WRITE_ERR_EID,          0x0000},
-    {NOVATEL_OEM615_UART_TIMEOUT_ERR_EID,        0x0000},
-    {NOVATEL_OEM615_MUT_CMD_DEVICE_ERR_EID,      0x0000},
-    {NOVATEL_OEM615_MUT_REQUEST_HK_ERR_EID,      0x0000},
-    {NOVATEL_OEM615_MUT_REQUEST_DATA_ERR_EID,    0x0000},
-    {NOVATEL_OEM615_MUT_INC_CMD_CNT_ERR_EID,     0x0000},
-    {NOVATEL_OEM615_MUT_INC_CMD_ERR_CNT_ERR_EID, 0x0000},
-    {NOVATEL_OEM615_MUT_INC_DEV_CNT_ERR_EID,     0x0000},
-    {NOVATEL_OEM615_MUT_INC_DEV_ERR_CNT_ERR_EID, 0x0000},
-    {NOVATEL_OEM615_MUT_GET_RUN_STATUS_ERR_EID,  0x0000},
-    {NOVATEL_OEM615_MUT_SET_RUN_STATUS_ERR_EID,  0x0000},
-    {NOVATEL_OEM615_MUT_GET_DEV_ENABLE_ERR_EID,  0x0000},
-    {NOVATEL_OEM615_MUT_SET_DEV_ENABLE_ERR_EID,  0x0000},
-    {NOVATEL_OEM615_CMD_LOG_INF_EID,             0x0000},
-    {NOVATEL_OEM615_CMD_UNLOG_INF_EID,           0x0000},
-    {NOVATEL_OEM615_CMD_UNLOGALL_INF_EID,        0x0000},
-    {NOVATEL_OEM615_CMD_SERIALCONFIG_INF_EID,    0x0000},
-
-    /* TODO: Add additional event IDs (EID) to the table as created */
-};
-
-
 /*
 ** Application entry point and main process loop
 */
 void NOVATEL_OEM615_AppMain(void)
```

Comment:
This name is too long to use in the .scr start script (limit of 20 chars, including null terminator).

Reply:
Should be resolved in commit https://github.com/nasa-itc/novatel_oem615/pull/7/commits/2d72932772288b3c6d5f2cf8fecd557f2a8b9f67